homeassistant/components/shelly/light.py

```diff
@@ -204,6 +217,30 @@ def supported_color_modes(self) -> set | None:
         """Flag supported color modes."""
         return self._supported_color_modes
 
+    @property
+    def effect_list(self):
+        """Lights RGB effects list."""
+        if self.wrapper.model == "SHBLB-1":
+            return list(SHBLB_1_RGB_EFFECTS.values())
+
+        return list(STANDARD_RGB_EFFECTS.values())
+
+    @property
+    def effect(self):
+        """Lights RGB current effect."""
```

Comment:
```suggestion
    def effect(self) -> str | None:
        """Return the current effect.""" 
```
Same here follow base component: [Base light effect](https://github.com/home-assistant/core/blob/8e38f26978073f18cc0bc59d9c8b6b917cb5c6e0/homeassistant/components/light/__init__.py#L674)